SConstruct

```diff
@@ -1246,6 +1254,55 @@ else:
 env['has_demangle'] = conf.CheckDeclaration("boost::core::demangle",
                                 '#include <boost/core/demangle.hpp>', 'C++')
 
+# check BLAS/LAPACK installations
+if env["blas_lapack_libs"]:
+    for lib in env["blas_lapack_libs"]:
+        if not conf.CheckLib(lib):
+            config_error(f"Library {lib!r} could not be found.")
+
+elif env["matlab_path"] != "" and env["matlab_toolbox"] == "default":
```

Comment:
Shouldn't this apply for either `default` or `y` values of the `matlab_toolbox` option?